lib/pharos/resources/openebs/22-storage-class.yml.erb

```diff
@@ -0,0 +1,10 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
```

Comment:
Maybe we should have option to set this as the default storage class.

Reply:
done